media-plugins/live/live-2020.03.06.ebuild

```diff
@@ -0,0 +1,130 @@
+}
+
+src_prepare() {
+	if use static-libs ; then
+		cp -pPR live live-shared
+		mv live live-static
+	else
+		mv live live-shared
+	fi
+
+	use static-libs && cp "${FILESDIR}/config.gentoo-r1" live-static/config.gentoo
+	cp "${FILESDIR}/config.gentoo-so-r2" live-shared/config.gentoo-so-r1
+
+	case ${CHOST} in
+		*-solaris*)
+			use static-libs && { sed -i \
+				-e '/^COMPILE_OPTS /s/$/ -DSOLARIS -DXLOCALE_NOT_USED/' \
+				-e '/^LIBS_FOR_CONSOLE_APPLICATION /s/$/ -lsocket -lnsl/' \
+				live-static/config.gentoo \
+				|| die ; }
+			sed -i \
+				-e '/^COMPILE_OPTS /s/$/ -DSOLARIS -DXLOCALE_NOT_USED/' \
+				-e '/^LIBS_FOR_CONSOLE_APPLICATION /s/$/ -lsocket -lnsl/' \
+				live-shared/config.gentoo-so-r1 \
+				|| die
+		;;
+		*-darwin*)
+			use static-libs && { sed -i \
+				-e '/^COMPILE_OPTS /s/$/ -DBSD=1 -DHAVE_SOCKADDR_LEN=1/' \
+				-e '/^LINK /s/$/ /' \
+				-e '/^LIBRARY_LINK /s/$/ /' \
+				-e '/^LIBRARY_LINK_OPTS /s/-Bstatic//' \
+				live-static/config.gentoo \
+				|| die static ; }
+			sed -i \
+				-e '/^COMPILE_OPTS /s/$/ -DBSD=1 -DHAVE_SOCKADDR_LEN=1/' \
+				-e '/^LINK /s/$/ /' \
+				-e '/^LIBRARY_LINK /s/=.*$/= $(CXX) -o /' \
+				-e '/^LIBRARY_LINK_OPTS /s:-shared.*$:-undefined suppress -flat_namespace -dynamiclib -install_name '"${EPREFIX}/usr/$(get_libdir)/"'$@:' \
+				-e '/^LIB_SUFFIX /s/so/dylib/' \
+				live-shared/config.gentoo-so-r1 \
+				|| die shared
+		;;
+	esac
+	multilib_copy_sources
+}
+
+src_configure() { :; }
+
+multilib_src_compile() {
+	tc-export CC CXX LD
+
+	if use static-libs ; then
+		cd "${BUILD_DIR}/live-static"
+
+		einfo "Beginning static library build"
+		./genMakefiles gentoo
+		emake -j1 LINK_OPTS="-L. $(raw-ldflags)"
+	fi
+
+	cd "${BUILD_DIR}/live-shared"
+	einfo "Beginning shared library build"
+	./genMakefiles gentoo-so-r1
+	local suffix=$(get_libname ${LIVE_ABI_VERSION})
+	emake -j1 LINK_OPTS="-lssl -L. ${LDFLAGS}" LIB_SUFFIX="${suffix#.}"
+
+	for i in liveMedia groupsock UsageEnvironment BasicUsageEnvironment ; do
+		pushd "${BUILD_DIR}/live-shared/${i}" > /dev/null
+		ln -s lib${i}.${suffix#.} lib${i}$(get_libname) || die
+		popd > /dev/null
+	done
+
+	if multilib_is_native_abi; then
+		einfo "Beginning programs build"
+		for i in $(use examples && echo "testProgs") proxyServer mediaServer ; do
+			cd "${BUILD_DIR}/live-shared/${i}"
+			emake LINK_OPTS="-L. ${LDFLAGS}"
+		done
+	fi
+}
+
+multilib_src_install() {
+	for library in UsageEnvironment liveMedia BasicUsageEnvironment groupsock; do
+		use static-libs && dolib.a live-static/${library}/lib${library}.a
+		dolib.so live-shared/${library}/lib${library}$(get_libname ${LIVE_ABI_VERSION})
+		dosym lib${library}$(get_libname ${LIVE_ABI_VERSION}) /usr/$(get_libdir)/lib${library}$(get_libname)
+
+		insinto /usr/include/${library}
+		doins live-shared/${library}/include/*h
+	done
+
+	if multilib_is_native_abi; then
+		# Should we really install these?
+		use examples && find live-shared/testProgs -type f -perm 755 -print0 | \
```

Comment:
(I personally think no, but in case you do, try accomplishing this using `-exec` instead of `xargs`. Append with `|| die`.

Reply:
I went for getting rid of them. People can let us know if they need them.